code/game/objects/items.dm

```diff
@@ -79,6 +79,10 @@
 
 /obj/item/New()
 	..()
+	if(force < 5) // Arbitrarily chosen
+		hitsound = 'sound/effects/slap1.ogg'
+	else
+		hitsound = 'sound/weapons/smash.ogg'
```

Comment:
```suggestion
		hitsound = force < 5 'sound/effects/slap1.ogg' : 'sound/weapons/smash.ogg'
```

Also maybe defines?

Reply:
You forgot a pretty important character in this suggestion

Reply:
true